presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/file/FileHiveMetastore.java

```diff
@@ -439,6 +429,23 @@ public synchronized void updatePartitionStatistics(HiveIdentity identity, Table
                 .collect(toImmutableList());
     }
 
+    @GuardedBy("this")
+    private List<String> listAllTables(String databaseName)
+    {
+        requireNonNull(databaseName, "databaseName is null");
```

Comment:
This check isn't needed, as we pass is straight to `getDatabase()` which already does the check.

Reply:
This is existing code and i prefer not to improve it this time.